### ISSUE TITLE
[request] Complete json setting 

### ISSUE THREAD

**Issue by nicofirst1** [alicevision/meshroom#683]
**Is your feature request related to a problem? Please describe.**
I would like to run the meshroom pipeline from command line using a custom setting in json format. The problem is that I have no clue on how the _settings.json_ should be constructed.

**Describe the solution you'd like**
A default _setting.json_ would be ideal to understand the structure or just to have a general overview of the parameters.

**Comment by natowi** [comment]
This might be interesting for you: http://filmicworlds.com/blog/command-line-photogrammetry-with-alicevision/ (use Alicevision Meshroom with .bat files)

**Comment by nicofirst1** [comment]
This is more of an alternative solution to my problem. 
I would rather run the the whole pipeline with _meshroom_photogrammetry_ modifying the _setting.json_ programmatically depending on the dataset.

**Comment by fabiencastan** [comment]
Yes, we should provide an exemple of setting.json in the manual.

**Comment by nicofirst1** [comment]
I wrote a semi-complete settings.json. Hope this helps

```
{
    "FeatureExtraction_1": {
        "describerPreset": "normal",
        "forceCpuExtraction": "0",
        "describerTypes": [
            "sift"
        ],
        "verboseLevel":"info"
    },
    "FeatureMatching_1": {
        "guidedMatching": "0",
        "describerTypes": [
            "sift"
        ],
        "verboseLevel":"info"

    },
    "StructureFromMotion_1": {
        "useLocalBA": "1",
        "lockAllIntrinsics": "0",
        "describerTypes": [
            "sift"
        ],
        "minInputTrackLength":"2",
        "lockScenePreviouslyReconstructed": "0",
        "maxNumberOfMatches": "0",
        "verboseLevel":"info"

    },
    "PrepareDenseScene_1": {
        "verboseLevel":"info"
    },
    "DepthMap_1": {
        "downscale": "2",
        "sgmMaxTCams": "10",
        "refineMaxTCams": "6",
        "verboseLevel":"info"
    },
    "DepthMapFilter_1": {
        "minNumOfConsistentCams": "3",
        "minNumOfConsistentCamsWithLowSimilarity": "4",
        "verboseLevel":"info"
    },

    "Meshing_1": {
        "maxInputPoints": "50000000",
        "maxPoints": "5000000",
        "estimateSpaceMinObservationAngle": "10",
        "colorizeOutput": "0",
        "verboseLevel":"info"

    },
    "MeshFiltering_1": {
        "removeLargeTrianglesFactor": "60",
        "keepLargestMeshOnly":"0",
        "iterations": "5",
        "verboseLevel":"info"

    },

    "Texturing_1": {
        "textureSide": "8192",
        "downscale":"1",
        "outputTextureFileType":"png",
        "unwrapMethod":"Basic",
        "useUDIM":"1",
        "fillHoles":"0",
        "verboseLevel":"info"

    }
    
}
```

**Comment by nicofirst1** [comment]
Are there any update on this?
Using the _setting.json_ file I provided overrides _every_ parameter.
So if you use a custom setting file without specifying all the possible parameters it will run the pipeline without them, thus getting stuck at featureExtraction.

**Comment by uberVelocity** [comment]
Are there any updates on this issue?

**Comment by KonstantinRr** [comment]
There does not to be a '.mg' file in this or the main AliceVision repository. Where is it located or are there any new updates on this issue?

**Comment by natowi** [comment]
@KonstantinRr You can save the default graph from the GUI

**Comment by nicofirst1** [comment]
Using the GUI default graph yields error for every key in the setting dictionary.
Seems like there is a big difference between the GUI setting and the mesh_compute one.

**Comment by natowi** [comment]
This might be useful:
#632

> meshroom_photogrammetry: "--save" does not disable the computation so you can save your batch reconstruction project and open it later interactively in meshroom. Additional --compute option if you need only to create a meshroom project file from this command line.
> 
> new positional argument to set project file or input folder
> new "--pipeline" option to override the default pipeline with your own project file. It also supports an environment variable MESHROOM_DEFAULT_PIPELINE.

**Comment by daphna-kaplan** [comment]
I am looking for an updated .json example of the setting file, trying to override using the data from the .mg file but I think somethink is wrong with my json structure.
